lucene/codecs/src/java/org/apache/lucene/codecs/simpletext/SimpleTextVectorReader.java

```diff
@@ -245,8 +245,8 @@ private void readAllVectors() throws IOException {
 
     private void readVector(float[] value) throws IOException {
       SimpleTextUtil.readLine(in, scratch);
-      // skip leading " [" and strip trailing "]"
-      String s = new BytesRef(scratch.bytes(), 2, scratch.length() - 3).utf8ToString();
+      // skip leading "[" and strip trailing "]"
+      String s = new BytesRef(scratch.bytes(), 1, scratch.length() - 2).utf8ToString();
```

Comment:
Wow, how did this ever work; we must never have tested it. grr. Thank you for cleaning up!